runtimes/peregrine/src/lib.rs

```diff
@@ -1166,7 +1166,9 @@ pub type Executive = frame_executive::Executive<
 	Runtime,
 	// Executes pallet hooks in the order of definition in construct_runtime
 	AllPalletsWithSystem,
-	(),
+	// Unfortunately, this migration fails the idempotency tests, and the `VersionedMigration` struct is still marked
+	// as experimental as of now.
+	pallet_assets::migration::v1::MigrateToV1<Runtime>,
```

Comment:
Why do we need the migration at all? We have no storage entries. I think if we just set the storage version, we are good to go.

Reply:
How would you do that? There's no extrinsic, so it would have to be a runtime migration. What this migration does, is to simply set the new storage version, since there's no other storage elements to migrate. Or maybe I am not getting your point?

Reply:
Yes, it would be a runtime migration to set up the storage version. We have this struct: https://github.com/KILTprotocol/kilt-node/blob/develop/runtimes/common/src/migrations.rs#L29 

Which can be utilized as: `pallet_assets::Pallet::<T>::current_storage_version().put::<pallet_assets::Pallet<T>>();`

Reply:
I mean, the provided migration does exactly the same. Why would we need a new custom migration just for that?

Reply:
We might get green CI flag for try-runtime tho. I will use this migration on Monday before merging, you convinced me.

Reply:
Yes okay. I also played a bit around, and I got the try-runtime tests passing by initializing the storage.

If you face this error: 

`[2024-07-26T14:19:04Z ERROR try-runtime::cli] failed to generate compact proof: TrieError(InvalidStateRoot(0x64c4b3358b6f671d81f8421100f5b73d1b1cbf931fc986ea8a0178db60cd7a0a))`

Update the try-runtime cli. There was a bug https://github.com/paritytech/try-runtime-cli/issues/76

Reply:
https://github.com/KILTprotocol/kilt-node/pull/676/commits/d2fcad5405d911cc9e024c7c8a8aa7c49a078807